Makefile

```diff
@@ -47,8 +47,11 @@ trim-dictionaries:
 check-manifest:
 	check-manifest --no-build-isolation
 
-check-distutils:
-	python setup.py check --restructuredtext --strict
+check-dist:
+	$(eval TMP := $(shell mktemp -d))
+	python -m build -o $(TMP)
```

Comment:
This requires [build](https://github.com/pypa/build), but build has not been added as a dependency in `pyproject.toml`.